docs/cli-help.txt

```diff
@@ -503,15 +503,6 @@ apache:
                         Full path to Apache control script (default:
                         apache2ctl)
 
-certbot-route53:auth:
```

Comment:
This file is automatically generated at release time and shouldn't be updated manually.